pkg/size/default.go

```diff
@@ -0,0 +1,460 @@
+package size
+
+import (
+	v1 "github.com/blackducksoftware/synopsys-operator/pkg/api/size/v1"
+	"github.com/blackducksoftware/synopsys-operator/pkg/apps/blackduck/types"
+	"github.com/blackducksoftware/synopsys-operator/pkg/util"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"strings"
+)
+
+// GetSize returns the size for each container
+func GetSize(name string) *v1.Size {
```

Comment:
Can we have the old way of getting GetSize at app level? I think this GetSize will be used as part of cmd_deploy command only where we can call the app level GetSize method to create their corresponding sizes. So that you will not have the circular dependency.. Is that sounds good?

Reply:
There is currently no circular dependency but moving GetSize into the app folder will create one